src/less/filter-menu-button/base/filter-menu-button.less

```diff
@@ -166,6 +158,11 @@ span.filter-menu-button__status--inline {
     }
 }
 
+span.filter-menu-button__status--background {
```

Comment:
Should we nest this so it looks like it's part of the filter-menu-button__status? I'm fine either way

IE
.filter-menu-button__status {
   // styles
   &--background {
   //More styles 
   }
}

Reply:
Ian has been asked that we don't nest, if possible. I'm fine to keep it separated.

Reply:
Ian recommends (and I agree with him) not doing this, because that makes it difficult to do a text search for a full class name, like `filter-menu-button__status--background`.